include/antler/project/dependency.hpp

```diff
@@ -65,17 +92,18 @@ class dependency {
 
    /// Get the github tag/commit hash.
    /// @return github tag/commit hash.
-   [[nodiscard]] std::string_view tag() const noexcept;
+   [[nodiscard]] const std::string& tag() const noexcept;
    /// Set github tag/commit hash.
    /// @param s  The new github tag/commit hash.
    void tag(std::string_view s) noexcept;
 
    /// Get the github release version.
    /// @return The possibly empty release version.
-   [[nodiscard]] std::string_view release() const noexcept;
+   [[nodiscard]] std::string release() const noexcept { return m_rel; }
```

Comment:
this file have mixed types, sometimes return value is `const std::string&`, sometimes `std::string` and sometimes `std::string_view`. It would be nice to make it consistent and to use only one unless there is reason for exception

Reply:
Good point, I agree, but probably not worth doing right now.